estate/demo/estate_property.xml

```diff
@@ -0,0 +1,81 @@
+<odoo>
+    <record id="umbm_estate_01" model="estate_property">
+        <field name="name">Big Villa 3</field>
+        <field name="state">new</field>
+        <field name="description">A nice and big villa</field>
+        <field name="postcode">12345</field>
+        <field name="estate_property_type" ref="estate.umbm_01"/>
+        <field name="date_availability">2024-04-25</field>
+        <field name="expected_price">1600000</field>
+        <field name="bedrooms">6</field>
+        <field name="living_area">100</field>
+        <field name="facades">4</field>
+        <field name="garage">True</field>
+        <field name="garden">True</field>
+        <field name="garden_area">100000</field>
+        <field name="garden_orentation">south</field>
+    </record>
+
+    <record id="umbm_estate_02" model="estate_property">
+        <field name="name">Trailer home 3</field>
+        <field name="state">canceled</field>
+        <field name="estate_property_type" ref="estate.umbm_01"/>
+        <field name="description">Home in a trailer park</field>
+        <field name="postcode">54321</field>
+        <field name="date_availability">2024-01-01</field>
+        <field name="expected_price">120000</field>
+        <field name="selling_price">120000</field>
```

Comment:
I'd say this shouldn't be defined here. What I like in demo data is to mimic the real flow. So the call to the method action_confirm is enough